packages/apps/spaces/hooks/useMeeting/useUpdateMeetingRecording.ts

```diff
@@ -0,0 +1,178 @@
+          if (participant?.__typename === 'UserParticipant') {
+            users.push(participant.userParticipant.id);
+          } else if (participant?.__typename === 'ContactParticipant') {
+            contacts.push(participant.contactParticipant.id);
+          }
+        }
+        request.append('users', JSON.stringify(users));
+        request.append('contacts', JSON.stringify(contacts));
+        request.append('topic', convert(response?.data?.meeting_Update.agenda));
+        request.append('type', 'meeting');
+        request.append('file_id', response?.data?.meeting_Update.recording);
+
+        axios
+        .post(`/transcription-api/transcribe`, request, {
+          headers: {
+            'accept': `application/json`,
+          },
+        })
+        .then((res) => {
+          if (res.status === 200) {
+                  toast.success(`Meeting recording updated successfully`, {
+                        toastId: `update-meeting-${meetingId}`,
+                  });
+          }
+        })
+      }
+      const data = client.cache.readFragment({
+        id: `Meeting:${meetingId}`,
+        fragment: gql`
+          fragment MeetingUpdateFragment on Meeting {
+            id
+            attendedBy {
+              ... on UserParticipant {
+                userParticipant {
+                  id
+                }
+              }
+              ... on ContactParticipant {
+                contactParticipant {
+                  id
+                }
+              }
+            }
+            meetingCreatedBy: createdBy {
+              ... on UserParticipant {
+                userParticipant {
+                  id
+                }
+              }
+              ... on ContactParticipant {
+                contactParticipant {
+                  id
+                }
+              }
+            }
+            startedAt
+            endedAt
+            createdAt
+            agenda
+            agendaContentType
+            recording
+          }
+        `,
+      });
+
+      if (data) {
+        client.cache.writeFragment({
+          id: `Meeting:${meetingId}`,
+          fragment: gql`
+            fragment MeetingUpdateFragment on Meeting {
+              id
+              attendedBy {
+                ... on UserParticipant {
+                  userParticipant {
+                    id
+                  }
+                }
+                ... on ContactParticipant {
+                  contactParticipant {
+                    id
+                  }
+                }
+              }
+              meetingCreatedBy: createdBy {
+                ... on UserParticipant {
+                  userParticipant {
+                    id
+                  }
+                }
+                ... on ContactParticipant {
+                  contactParticipant {
+                    id
+                  }
+                }
+              }
+              startedAt
```

Comment:
meetingStartedAt: startedAt
meetingEndedAt: endedAt

we are aliasing those two